.github/workflows/tests_cpu.yml

```diff
@@ -0,0 +1,41 @@
+on: [push, pull_request]
+
+jobs:
+  test-cpu:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
+        pytorch-channel: [pytorch, pytorch-nightly]
+        include:
+        - os: windows-latest
+          pytorch-channel: pytorch
+          python-version: 3.7
```

Comment:
Why test against `3.7` for macos and windows? Why not `3.6`?

Reply:
no specific reason

Reply:
Given that you test `3.[6-8]` on Ubuntu, I would only test the oldest version as Python is generally backwards compatible. Otherwise you don't know if `3.6` causes problems on macOS or Windows.